auth_otp/serializers.py

```
@@ -0,0 +1,375 @@
+"""
+سریالایزرهای سیستم احراز هویت OTP
+OTP Authentication System Serializers
+"""
+
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+from django.core.validators import RegexValidator
+from .models import OTPRequest, OTPVerification, TokenBlacklist
```

Ruff (0.12.2)

`.models.OTPVerification` imported but unused

Remove unused import

(F401)